MainContainer/config/db.js

```diff
@@ -7,10 +7,10 @@
   Warning: Caminte has not been updated - some adapters are not properly working.
   We can confirm that MySQL and Mongo switches work. 
 */
-const caminte = require("caminte");
-const Schema = caminte.Schema;
+var caminte = require("caminte");
+var Schema = caminte.Schema;
 
-const config = {
+var config = {
   driver: "mysql",
   host: "localhost",
   username: "wapataho_vpia",
```

Comment:
Why have these changed from `const` to `var`? Is it due to your branch being older than develop?